kernel/init.c

```diff
@@ -37,6 +37,9 @@
 #include <zephyr/logging/log.h>
 #include <zephyr/pm/device_runtime.h>
 #include <zephyr/internal/syscall_handler.h>
+#ifdef CONFIG_BOOTARGS
+#include <ctype.h>
+#endif /* CONFIG_BOOTARGS*/
```

Comment:
No need for conditional inclusion of this header.
```suggestion
#include <ctype.h>
```

Reply:
The guarding ifdef was removed and this include was moved before Zephyr’s includes. At the same time other standard includes were moved too, to keep all standard includes in front of Zephyr ones